test/testenv.py

```diff
@@ -104,8 +104,7 @@ def run_test(self):
 
         assert output_docs, 'empty output'
         assert expect_docs, 'empty expected'
-        assert output_docs == expect_docs
-        assert output_errors == expect_errors
+        assert output_docs + output_errors == expect_docs + expect_errors
```

Comment:
Is it worth to `'\n'.join(...)` to improve the diff or is `output_docs` guaranteed to end in a new line?

Reply:
Per my testing, there's always been a newline, and the diff has looked fine with this. (And I've looked at plenty when trying to debug the macos CI issues.)